File: bin/convert-btc-to-usd.js
#!/usr/bin/node

if (process.argv.length !== 3) {
    console.log('Pass BTC amount as parameter');
    process.exit(1);
}

(async () => {
    const resp = await fetch('https://api.coinbase.com/v2/prices/BTC-USD/buy');
    const data = await resp.json();
    const usdToBtc = data['data']['amount'];
    const formatter = new Intl.NumberFormat('en-US', {
        style: 'currency',
        currency: 'USD'
    });
    console.log(formatter.format(usdToBtc * process.argv[2]));
})();

File: bin/convert_btc_to_usd.py
#!/usr/bin/env python3

from json import loads
from requests import get
from sys import argv, stderr, exit

def convert_btc_to_usd(btc_value):
    resp = get("https://api.coinbase.com/v2/prices/BTC-USD/buy")
    if resp.status_code == 200:
        resp_json = loads(resp.text)
        usdToBtc = float(resp_json["data"]["amount"])
        return usdToBtc * btc_value
    else:
        print("Coinbase request failed", stderr)

if __name__ == "__main__":
    if (len(argv) != 2):
        print("Pass BTC amount as parameter")
        exit(1)

    print(f"${convert_btc_to_usd(float(argv[1])):,.2f}")
